src/recipe.jl

```diff
@@ -110,6 +113,53 @@ function Makie.plot!(plot::Beeswarm)
         calculate!(point_buffer.val, algorithm, direction == :y ? pixelspace_point_buffer.val : reverse.(pixelspace_point_buffer.val), markersize, side)
         # Project the beeswarm back to data space and store it, again, in `point_buffer.val`
         point_buffer.val .= Point2f.(Makie.project.((scene.camera,), :pixel, :data, direction == :y ? (point_buffer.val) : reverse.(point_buffer.val)))
+
+        # Method to create a gutter when a gutter is defined
+        # NOTE: Maybe turn this into a helper function?
+
+        if !isnothing(plot.attributes[:gutter].val)
+            # Get category labels
+            groups = [pt[1] for pt in positions] 
+
+            # Sort positions before iterating through values
+            sort!([positions], by = x -> x[1])
+
+            # Find all points belonging to all unique categories
+            idx = 1
+            for group in unique(groups) |> sort
+                
+                # Starting index for the group
+                starting = findfirst(==(group), groups)
+
+                # Last index for the group
+                ending = findlast(==(group), groups)
+
+                # Calculate a gutter threshold
+                gutter_threshold_count = (ending - starting) * plot.attributes[:gutter_threshold].val
```

Comment:
`gutter` and `gutter_threshold` should also be in the `onany` expression, so that the plot responds to updates.

Reply:
Got it done!